[tutorial] Ch 3-6 questions redone (#1852)
withastro/docs: src/pages/en/tutorial/3-components/1.md

### Test your knowledge

> 1. Astro files can use these in their template:

Author: Wait, did I not reduce this to three?

> 1. Astro files can use these in their template:

Reviewer: Three options but still four questions!

> 1. Astro files can use these in their template:

Author: lol, I reduced to three OPTIONS but then didn't get rid of a question!

> 1. Astro files can use these in their template:

Author: ```suggestion
```

>   <MultipleChoice>

Reviewer: Oh and a bunch more indentation fixes:
```suggestion
    <MultipleChoice>
```

    <Option>
      standard HTML elements
    </Option>
    <Option>
      imported Astro components
    </Option>
    <Option isCorrect>

>        both of the above, together!

Author: What do you think about "both of the above, alongside each other!" (That more closely mimics the older content, and gets the idea across that they can occur alongside each other.)

>     <MultipleChoice>
>     <Option>
>       standard HTML elements
>     </Option>
>     <Option>
>       imported Astro components
>     </Option>
>     <Option isCorrect>
>        both of the above, together!
>     </Option>
>   </MultipleChoice>

Author: ```suggestion
```

> 2. You can do this when you have elements repeated on multiple pages:

Author: ```suggestion
1. You can do this when you have elements repeated on multiple pages:
```

>   <MultipleChoice>

Reviewer: ```suggestion
    <MultipleChoice>
```

      <Option>
        restart the dev server
      </Option>
      <Option isCorrect>
        refactor to use a reusable component
      </Option>
      <Option>
        make a new page
      </Option>
    </MultipleChoice>

> 3. Astro components are:

Author: ```suggestion
2. Astro components are:
```

>   <MultipleChoice>

Reviewer: ```suggestion
    <MultipleChoice>
```

> 2. You can do this when you have elements repeated on multiple pages:
>   <MultipleChoice>
>       <Option>
>         restart the dev server
>       </Option>
>       <Option>
>         refactor
>       </Option>
>       <Option isCorrect>
>         make a new page
>       </Option>
>     </MultipleChoice>
> 
> 3. Astro components are:
>   <MultipleChoice>
>     <Option isCorrect>
>       reusable
>     </Option>
>     <Option>
>       fragments of HTML
>     </Option>
>     <Option>
>        both of the above!
>     </Option>
>   </MultipleChoice>

Author: ```suggestion
2. You can do this when you have elements repeated on multiple pages:
  <MultipleChoice>
      <Option>
        restart the dev server
      </Option>
      <Option isCorrect>
        refactor
      </Option>
      <Option>
        make a new page
      </Option>
    </MultipleChoice>

3. Astro components are:
  <MultipleChoice>
    <Option>
      reusable
    </Option>
    <Option>
      fragments of HTML
    </Option>
    <Option isCorrect>
       both of the above!
    </Option>
  </MultipleChoice>
```

> 4. Astro components will automatically create a new page on your site when you...

Author: ```suggestion
3. Astro components will automatically create a new page on your site when you...
```

>   <MultipleChoice>

Reviewer: ```suggestion
    <MultipleChoice>
```

### Test your knowledge

> 1. Astro files can use these alongside Astro components in their template:
>   <MultipleChoice>
>     <Option>
>       reusable
>     </Option>
>     <Option>
>       `src/pages/`
>     </Option>
>     <Option>
>        refactor
>     </Option>
>     <Option isCorrect>
>        standard HTML elements
>     </Option>
>   </MultipleChoice>

Reviewer: Trimming down to 3 options and… maybe being too silly with the options?

```suggestion
1. Astro files can use these alongside Astro components in their template:
    <MultipleChoice>
      <Option>
        too much JavaScript
      </Option>
      <Option>
        `src/pages/`
      </Option>
      <Option isCorrect>
        standard HTML elements
      </Option>
    </MultipleChoice>
```

Author: I like being silly! But, they probably CAN use too much JS?  😅 

I'll think of another silly one!

> 2. You can do this when you have elements repeated on multiple pages:
>   <MultipleChoice>
>       <Option>
>         reusable
>       </Option>
>       <Option>
>         `src/pages/`
>       </Option>
>       <Option isCorrect>
>         refactor
>       </Option>
>       <Option>
>         standard HTML elements
>       </Option>
>     </MultipleChoice>

Reviewer: 4 questions with 4 options each seems a lot to me and 2 & 3 here are very similar. How do you feel about condensing to 3 questions with 3 options each?

```suggestion
2. You can do this when you have elements repeated on multiple pages:
    <MultipleChoice>
      <Option>
        copy/paste each time you need the elements
      </Option>
      <Option isCorrect>
        refactor to use reusable components
      </Option>
      <Option>
        use `<style>` to change their appearance
      </Option>
    </MultipleChoice>
```

> 
> 3. Astro components give you fragments of HTML that are:
>   <MultipleChoice>
>     <Option isCorrect>
>       reusable
>     </Option>
>     <Option>
>       `src/pages/`
>     </Option>
>     <Option>
>        refactor
>     </Option>
>     <Option>
>        standard HTML elements
>     </Option>
>   </MultipleChoice>

Reviewer: I’d remove this one as too similar to the previous one.

```suggestion
```

> 4. Unless you want to make a new page on your site, don't put your components here:
>   <MultipleChoice>
>     <Option>
>       reusable
>     </Option>
>     <Option isCorrect>
>       `src/pages/`
>     </Option>
>     <Option>
>        refactor
>     </Option>
>     <Option>
>        standard HTML elements
>     </Option>
>   </MultipleChoice>

Reviewer: This one feels like it might benefit from more similar options to force the reader to think a bit more — not sure how you feel about that.

```suggestion
4. Unless you want to make a new page on your site, don't put your components here:
  <MultipleChoice>
    <Option>
      `src/layouts/`
      <Fragment slot="feedback">
         Try again! A `layouts` directory is a good place to put any layout components you build and won’t create new pages on your site.
       </Fragment>
    </Option>
    <Option isCorrect>
      `src/pages/`
    </Option>
    <Option>
       `src/components/`
       <Fragment slot="feedback">
         Try again! A `components` directory is a good place for your components to live and won’t create new pages on your site.
       </Fragment>
    </Option>
  </MultipleChoice>
```

Author: I also like this, but we haven't done layouts yet! So, I'll work on this in the same vein.
